test/test_configuration.rb

```diff
@@ -20,15 +20,6 @@ class TestConfiguration < JekyllUnitTest
       assert_equal result["source"], "blah"
     end
 
-    should "fix common mistakes" do
```

Comment:
Just have a test here to ensure it's there and returns a valid `Configuration` instance.